ja_jp/contacts-manage-groups.adoc

```diff
@@ -0,0 +1,92 @@
+=== 連絡先リストを管理する
+
+連絡先を含むリストを作成し、そのリストを *連絡先* フォルダの下に保存することができます。
+これは、特定の人たちにメールを送りたいときに特に便利です。
+これは、全員のメールアドレスを入力するのではなく、リストの名前を入力すると、{product-short}はそのリストに含まれるすべてのアドレスにメールを送信します。
+
+連絡先をグループ化する場合、以下の操作が必要となります。
+
+. リストを作成する
+. 作成したリストへ連絡先を追加する
+
+==== 連絡先リストを作成する
+. btn:[連絡先] タブを選択します。
+. *新しいリスト* に、連絡先リスト名を入力します。
+. kbd:[Enter] を押し、リストを保存します。
+
+作成したリストを右クリックすることで、リスト名の変更、またはリストの削除が可能です。
+
+==== リストに連絡先を追加する
+. 連絡先のフォルダ名をクリックし、保存している連絡先を表示します。
+. リストへ追加希望の連絡先を選択します。
+. 新しく作成したリストへ選択した連絡先をドラッグアンドドロップします。

```

Comment:
Drag&Drop does not work

Reply:
I will remove this line as this feature has not been implemented yet.